deployments/engine/values.yaml

```diff
@@ -31,3 +31,8 @@ ingress:
   ingressClassName:
 
 version:
+
+scaledobject:
+  minReplicaCount: 1
+  maxReplicaCount: 2
+  prometheusServerAddr: http://prometheus-server.monitoring.svc.cluster.local
```

Comment:
I might later move this definition to https://github.com/llm-operator/llm-operator/blob/main/hack/llm-operator-values.yaml